CMakeLists.txt

```diff
@@ -206,6 +206,9 @@ if (CONFIG_BUILD_WITH_TFM)
     set(TFM_IMAGE_NUMBER_ARG
         MCUBOOT_IMAGE_NUMBER ${CONFIG_TFM_MCUBOOT_IMAGE_NUMBER})
   endif()
+  if (DEFINED CMAKE_BUILD_TYPE)
+    set(BUILD_TYPE_ARG CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE})
```

Comment:
See comment here: https://github.com/zephyrproject-rtos/trusted-firmware-m/pull/44#discussion_r570518633

Is there a special NCS use-case and is it needed for NCS 1.5 ?

Reply:
I was using it to get the RAM usage down to fit with the modem RAM. It turned out not to be needed, but might as well add it now that I completed it.